pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
Oh, I think we should not introduce this kind of cryptic profile name. Could you name it something like "sigmah-handicap-international"? It would be less confusing for the random developer.